inlong-manager/manager-pojo/src/main/java/org/apache/inlong/manager/pojo/sink/kafka/KafkaSink.java

```diff
@@ -61,6 +61,18 @@ public class KafkaSink extends StreamSink {
     @ApiModelProperty("Primary key is required when serializationType is json, avro")
     private String primaryKey;
 
+    @ApiModelProperty("the database-table mapping rule")
+    private String topicPattern;
+
+    @ApiModelProperty(value = "Automatically create kafka topic or not, a note for users.")
+    private String autoCreateTopics;
+
+    @ApiModelProperty(value = "the partition strategy for kafka")
+    private String partitionStrategy;
+
+    @ApiModelProperty("data multiple format,only applicable when outer format is raw")
```

Comment:
What does the `data multiple format` mean?

Reply:
for debezium and canal, the outer format is "raw" , and then sinkmultipleformat is "debezium"/"canal"

Reply:
used by sort connector to parse debezium/canal json

Reply:
What is the outer format? I didn't see it in KafkaSink.